signin-client.html

```diff
@@ -25,8 +25,8 @@
     <main>
         <section class="signup-box">
             <h1>Client Sign In</h1>
-            <form action="customer-dashboard.html" method="GET" onsubmit="saveCustomer()"> 
-                
+            <div id="error" style="color: red;"></div>
+            <form id="client-sign-in-form">
                 <!--Username-->
```

Comment:
does it save the customer when logging in and we go into another header

Reply:
It does because I call the saveCustomer() function on my javascript. I think it's better not to use method routes like GET or POST right now as that will be more necessary for backend